src/main/java/core/gui/comp/table/HOTableModel.java

```diff
@@ -454,4 +455,13 @@ private boolean setRowOrderSettings(RowSorter<HOTableModel> sorter) {
 		}
 		return changed;
 	}
+
+	/**
+	 * Returns the primary table
```

Comment:
I must admit I don't quite get the intent here.  I understand the model is attached to a list of tables, but I don't follow that idea of a primary table?

Reply:
Until now i found one case, where the same model is used in two different tables. It is in the transfer panel. 
If one of those tables will change the setting (column order, width or row order) it will be used for both tables. If the user will change both table settings of the transfer pane, the first of them is the winner.

All other usages of the HOTableModel will be used only by one single table (i called it the primary table).

An alternative solution to this would be to create two independant models for these transfer tables.